### ISSUE TITLE
Doesn't work in devcontainer

### ISSUE THREAD

**Issue by larsnystrom** [msamgan/vscode-larapint#25]
When I try to use this extension in a devcontainer, an error occurs:

```
Something went wrong while running Laravel Pint In ConfigurationResolver.php line 390: The path "/workspaces/myapp/app/Exceptions/Handler.php" is not readable.
```

I suspect the extension doesn't execute in the context of the devcontainer? I'm not sure how extensions works, but the file exists and is readable within the container.

Thank you!

**Comment by msamgan** [comment]
thanks for the issue.
ill look into it.

**Comment by larsnystrom** [comment]
Hi again, it appears this was a problem on my end. I can now run the "Laravel Pint: Format current file" command. However, I can't get the regular "Format Document" command to work. I have added 
```
  "[php]": {
    "editor.defaultFormatter": "msamgan.laravel-pint-vscode"
  },
```

to my settings.json, but when I run the "Format Document" command, it says "There is no formatter for 'php' files installed.".

**EDIT:** When I open my workspace using Remote-SSH and try to "Format Document" it says "Extension 'Laravel Pint Formatter' is configured as formatter but it cannot format 'PHP' files". Which is strange?

**EDIT2:** I have also added `"editor.laravel.pint.enabled": true` to my settings.json, and still experience the same issue. However, that setting is grayed out so maybe it has changed name or something?
<img width="625" alt="Skärmavbild 2023-03-15 kl  09 47 20" src="https://user-images.githubusercontent.com/1000370/225255801-38e65e3c-bd11-4c67-b405-411fa86e688d.png">
